packages/database/src/core/AuthTokenProvider.ts

```diff
@@ -104,3 +111,13 @@ export class AuthTokenProvider {
     warn(errorMessage);
   }
 }
+
+/* Auth token provider used to connect to the Emulator. */
+export class EmptyAuthTokenProvider implements AuthTokenProvider {
+  getToken(forceRefresh: boolean): Promise<FirebaseAuthTokenData> {
+    return Promise.resolve({ accessToken: '' });
```

Comment:
@yuchenshi can comment but I think the Admin SDKs are supposed to send the special token `owner` when talking to the emulators, rather than blank.